File: app/src/main/java/hu/andras/daggersample/di/Feature2ListSubcomponent.java
package hu.andras.daggersample.di;

import dagger.Subcomponent;
import hu.andras.daggersample.ui.feature2.list.Feature2ListActivity;

/**
 * Created by Andras Nemeth on 2017. 06. 14..
 */

@ActivityScope
@Subcomponent(modules = Feature2Module.class)
public interface Feature2ListSubcomponent {

    void inject(Feature2ListActivity feature2ListActivity);

}

File: app/src/main/java/hu/andras/daggersample/di/Feature2ListSubcomponent.kt
package hu.andras.daggersample.di

import dagger.Subcomponent
import hu.andras.daggersample.di.scopes.ActivityScope
import hu.andras.daggersample.ui.feature2.list.Feature2ListActivity

/**
 * Created by Andras Nemeth on 2017. 06. 14..
 */
@ActivityScope
@Subcomponent(modules = [Feature2Module::class])
interface Feature2ListSubcomponent {
    fun inject(feature2ListActivity: Feature2ListActivity?)
}
